flight/flight-core/src/main/java/org/apache/arrow/flight/grpc/NettyClientBuilder.java

```diff
@@ -130,6 +131,11 @@ public NettyClientBuilder verifyServer(boolean verifyServer) {
     return this;
   }
 
+  public NettyClientBuilder userAgent(String userAgent) {
```

Comment:
I'm against adding to this interface - the intent is that for any further customization, you should directly use the returned NettyChannelBuilder